src/api/tokio.rs

```diff
@@ -96,21 +96,7 @@ impl ApiBuilder {
     /// ```
     pub fn new() -> Self {
```

Comment:
It could be nice to have the token as an optional argument

Reply:
It's already there in the builder actually.